app/assets/stylesheets/application.css

```diff
@@ -32,6 +32,15 @@
   font-family: serif;
 }
 
+.table-light {
```

Comment:
Did the BS table-light class not work?

Reply:
It didn't work for me, and neither did BS table-bordered.  Not sure why, but perhaps someone would want to take another look at my CSS.